test/test.js

```diff
@@ -226,3 +226,45 @@ describe('setting up an initializing a local node', function () {
     })
   })
 })
+
+describe('change config values of a disposable node', function () {
+  this.timeout(20000)
```

Comment:
random arbitrary number? maybe define it somewhere and explain why the value was chosen?